### ISSUE TITLE
arguments evaluated for logs below current log level

### ISSUE THREAD

**Issue by erjoalgo** [orthecreedence/vom#5]
It is possible that arguments to logs below the current level not be evaluated?

```
> vom:*config*
(T :INFO)
> (vom:debug "~A" (progn (format t "args evaluated~%" ) '(1 2 3)))
args evaluated
NIL
> (vom:info "~A" (progn (format t "args evaluated~%" ) '(1 2 3)))
args evaluated
  <INFO> [19:20:25] my-package - (1 2 3)

```

**Comment by ANDRON94** [comment]
@erjoalgo No, it isn't possible in the current implementation of 'define-level' macro.
The macros are expanded on compile-time, so to disable evaluation of arguments you should check the log level on compile-time too. And it means that if you want to print some debug logs you should recompile whole your program.
Pros: Only relevant code is compiled and then evaluated.
Cons: You can't dynamically change the log level.

I don't know is it the useful feature but if it is it should be implemented with some kind of flag that allows the user to enable or disable such behavior.
